diff -Naur vlc-3.0-3.0.12.1.old/configure.ac vlc-3.0-3.0.12.1/configure.ac
--- vlc-3.0-3.0.12.1.old/configure.ac	2021-01-06 08:48:11.178012965 +0100
+++ vlc-3.0-3.0.12.1/configure.ac	2021-01-06 10:47:19.830919131 +0100
@@ -3479,14 +3479,14 @@
     [Multi-Media Abstraction Layer (MMAL) hardware plugin (default enable)]))
 if test "${enable_mmal}" != "no"; then
   VLC_SAVE_FLAGS
-  LDFLAGS="${LDFLAGS} -L/opt/vc/lib -lvchostif"
-  CPPFLAGS="${CPPFLAGS} -isystem /opt/vc/include -isystem /opt/vc/include/interface/vcos/pthreads -isystem /opt/vc/include/interface/vmcs_host/linux"
+  LDFLAGS="${LDFLAGS} -L/usr/lib64/ -lvchostif"
+  CPPFLAGS="${CPPFLAGS} -isystem /usr/include/vc -isystem /usr/include/vc/interface/vcos/pthreads -isystem /usr/include/vc/interface/vmcs_host/linux"
   AC_CHECK_HEADERS(interface/mmal/mmal.h,
-    [ AC_CHECK_LIB(bcm_host, vc_tv_unregister_callback_full, [
+    [ AC_CHECK_LIB(bcm_host, [
         have_mmal="yes"
         VLC_ADD_PLUGIN([mmal])
-        VLC_ADD_LDFLAGS([mmal],[ -L/opt/vc/lib ])
-        VLC_ADD_CFLAGS([mmal],[ -isystem /opt/vc/include -isystem /opt/vc/include/interface/vcos/pthreads -isystem /opt/vc/include/interface/vmcs_host/linux ])
+        VLC_ADD_LDFLAGS([mmal],[ -L/usr/lib64/ ])
+        VLC_ADD_CFLAGS([mmal],[ -isystem /usr/include/vc -isystem /usr/include/vc/interface/vcos/pthreads -isystem /usr/include/vc/interface/vmcs_host/linux ])
         VLC_ADD_LIBS([mmal],[ -lbcm_host -lmmal -lmmal_core -lmmal_components -lmmal_util -lvchostif ]) ], [
           AS_IF([test "${enable_mmal}" = "yes"],
             [ AC_MSG_ERROR([Cannot find bcm library...]) ],
