diff -Naur vlc-3.0.18.old/modules/access/dvdnav.c vlc-3.0.18/modules/access/dvdnav.c
--- vlc-3.0.18.old/modules/access/dvdnav.c	2022-10-25 13:46:10.582638329 +0200
+++ vlc-3.0.18/modules/access/dvdnav.c	2022-10-25 13:47:30.781066163 +0200
@@ -376,14 +376,7 @@
         goto bailout;
 
     /* Open dvdnav */
-#if DVDREAD_VERSION < DVDREAD_VERSION_CODE(6, 1, 2)
-    /* In libdvdread prior to 6.1.2, UTF8 is not supported for windows and
-     * requires a prior conversion.
-     * For non win32/os2 platforms, this is just a no-op */
     psz_path = ToLocale( psz_file );
-#else
-    psz_path = psz_file;
-#endif
 #if DVDNAV_VERSION >= 60100
     dvdnav_logger_cb cbs;
     cbs.pf_log = DvdNavLog;
@@ -402,10 +395,8 @@
 
 bailout:
     free( psz_file );
-#if DVDREAD_VERSION < DVDREAD_VERSION_CODE(6, 1, 2)
     if( psz_path )
         LocaleFree( psz_path );
-#endif
     return i_ret;
 }
 
diff -Naur vlc-3.0.18.old/modules/access/dvdread.c vlc-3.0.18/modules/access/dvdread.c
--- vlc-3.0.18.old/modules/access/dvdread.c	2022-10-25 13:46:10.582638329 +0200
+++ vlc-3.0.18/modules/access/dvdread.c	2022-10-25 13:47:30.781066163 +0200
@@ -212,14 +212,7 @@
     }
 
     /* Open dvdread */
-#if DVDREAD_VERSION < DVDREAD_VERSION_CODE(6, 1, 2)
-    /* In libdvdread prior to 6.1.2, UTF8 is not supported for windows and
-     * requires a prior conversion.
-     * For non win32/os2 platforms, this is just a no-op */
     const char *psz_path = ToLocale( psz_file );
-#else
-    const char *psz_path = psz_file;
-#endif
 #if DVDREAD_VERSION >= DVDREAD_VERSION_CODE(6, 1, 0)
     dvd_logger_cb cbs;
     cbs.pf_log = DvdReadLog;
@@ -227,9 +220,7 @@
 #else
     dvd_reader_t *p_dvdread = DVDOpen( psz_path );
 #endif
-#if DVDREAD_VERSION < DVDREAD_VERSION_CODE(6, 1, 2)
     LocaleFree( psz_path );
-#endif
     if( p_dvdread == NULL )
     {
         msg_Err( p_demux, "DVDRead cannot open source: %s", psz_file );
